common/v2/features/DevTools/ToolsAccountList.tsx

```diff
@@ -22,12 +23,18 @@ export interface AccountListProps {
 
 const ToolsAccountList: React.FC<AccountListProps> = props => {
   const { accounts, deleteAccount } = props;
-  const list = accounts.map((account: ExtendedAccount) => (
-    <AccountContainer>
-      <Address title={account.label} address={account.address} truncate={truncate} />
-      <DeleteButton onClick={() => deleteAccount(account.uuid)} icon="exit" />
-    </AccountContainer>
-  ));
+  let key = 0;
+  const list = accounts.map((account: ExtendedAccount) => {
+    key += 1;
+    const detectedLabel: AddressBook | undefined = getLabelByAccount(account);
+    const label = !detectedLabel ? 'Unknown Account' : detectedLabel.label;
+    return (
+      <AccountContainer key={key}>
```

Comment:
the second argument of `map` is the index of the current elem. You can use this as the key.
```
const list = accounts.map((account, idx) => {
  //  ...
   <AccountContainer key={idx}>
  //...
}